packages/toc/style/base.css

```diff
@@ -20,8 +20,23 @@
   height: 100%;
 }
 
+.jp-TableOfContents-placeholder {
+  text-align: center;
+}
+
+.jp-TableOfContents-placeholderContent {
+  color: var(--jp-content-font-color2);
+  padding: 8px;
+}
+
+.jp-TableOfContents-placeholderContent h3 {
+  margin-bottom: var(--jp-content-heading-margin-bottom);
+}
+
 .jp-TableOfContents .jp-SidePanel-content {
   overflow-y: auto;
+  display: flex;
+  align-items: flex-start;
```

Comment:
This breaks the execution indicator - I guess it should only be applied in the case of the placeholder

Reply:
These two lines weren't necessary for the placeholder; I've removed them.